services/echo/libecho/src/migrations/diesel/sqlite/src/schema.rs

```diff
@@ -0,0 +1,46 @@
+table! {
```

Comment:
Is this schema.rs file required? there is another one added in a later commit that is not sqlite specific

Reply:
It is not required and I thought I already deleted that, should be gone now